Misc/NEWS.d/next/Library/2022-04-09-11-28-49.bpo-34480.Pw-GJ6.rst

```diff
@@ -0,0 +1 @@
+Fix a bug where `match` variable is used prior to being defined when an invalid markup is encountered and :meth:`ParserBase.error` is implemented as a NOP in a child class of :class:`ParserBase`.
```

Comment:
Please rewrite the NEWS entry. It should be formulated in terms understandable for common user. Reference to variable is a reference to internal details about which a common user know nothing.

The issue was that in some circumstances you get an UnboundVariableError. The fix is that you no longer get an UnboundVariableError. Describe these circumstances and behavior changes.